tasks/views.py

```diff
@@ -1,3 +1,33 @@
 # from django.shortcuts import render
+from django.shortcuts import redirect, render
+from tasks.forms import TaskForm
+from tasks.models import Task
+from django.contrib import messages
+
 
 # Create your views here.
+def new_task(request):
+    if request.method == "POST":
+        form = TaskForm(request.POST)
+        if form.is_valid():
+            taskForm = form.save(commit=False)
+            try:
+                Task.create_task(taskForm.title, taskForm.assignee, taskForm.created_by, taskForm.priority,
+                                 taskForm.status,
+                                 taskForm.description)
+                messages.success(request, 'Task was added successfully')
+                form = TaskForm()
+                redirect('new_task')
+            except Exception as e:
+                messages.warning(request, e)
+                form = TaskForm(
+                    initial={'title': taskForm.title, 'assignee': taskForm.assignee, 'created_by': taskForm.created_by,
+                             'priority': taskForm.priority,
+                             'status': taskForm.status, 'description': taskForm.description})
+
+    else:
+        form = TaskForm()
+
+    return render(request, 'new_task.html', {
+        'form': form
+    })
```

Comment:
Please do it in one line:
```suggestion
    return render(request, 'new_task.html', {'form': form})
```